tern/analyze/options.py

```diff
@@ -0,0 +1,12 @@
+class Options:
+    '''A representation of user options
+    attributes:
+        shell: the shell found in the docker container
+        redo: redo flag to repopulate the cache for found layers
+        driver: required argument when running Tern in a container. If no
+        input is provided, 'fuse' will be used as the default option.
+    '''
+    def __init__(self, shell, redo, driver=None):
```

Comment:
An issue I encoutered earlier was when I added more options I ended up with functions with too many arguments (linter complained). If there would be more args added later, someone would encounter the same problem. What do you think about creating instance of this class in __main__.py and letting argparse set members (via https://docs.python.org/3/library/argparse.html#dest) rather than via __init__()?

Reply:
Having the `Options` class in `__main__.py` and having argparse set members is how I would expect this to work.

Reply:
@rheo-chiti -- can you move the `Options` class to __main__.py and use argparse to set the members?

Reply:
@nishakm @rnjudge  , I moved the ```Options``` class to main.py but I am still unable to understand how to use argparse to set it's members.

Reply:
I can take a look later...